src/main/kotlin/com/personio/synthetics/client/BrowserTest.kt

```diff
@@ -12,10 +12,14 @@ import com.personio.synthetics.config.loadConfiguration
 
 /**
  * Creates a synthetic browser test with the added configurations (if any) and steps in Datadog
- * The default configurations for the tests are loaded from the configuration file in resources
+ * The default configurations for the tests are loaded from the configuration file in resources.
+ *
+ * Deprecation notice: In future releases this method will be replaced by one that
+ * uses SyntheticTestBuilder or similar.
  * @param name Name of the test
  * @param steps Calls the added steps and configuration functions of the test
  */
+@Deprecated(message="Planned for removal in 3.x.x")
```

Comment:
Aren't we gonna keep this method but only move and rewrite it?

Reply:
Yes, but the API signature and package of this method will change.

`BrowserTest` as it stands. Should I change the message to something along the lines of: `Will have its signature and package changed on 3.x.x`?

Reply:
Rephrased the warning, could you please take another look